framework/test/jest/unit/specs/modules/chain/synchronizer/block_synchronization_mechanism.spec.js

```diff
@@ -0,0 +1,183 @@
+const BlockSynchronizationMechanism = require('../../../../../../../src/modules/chain/synchronizer/block_synchronization_mechanism');
+
+describe('BlockSynchronizationMechanism', () => {
+	const stubs = {
+		channel: {
+			invoke: jest.fn(),
+		},
+		modules: {
+			blocks: {
+				lastBlock: jest.fn(),
+			},
+		},
+	};
+	let blockSynchronizationMechanism;
+
+	beforeEach(() => {
+		blockSynchronizationMechanism = new BlockSynchronizationMechanism({
+			channel: stubs.channel,
+			modules: stubs.modules,
+		});
+	});
+
+	describe('constructor()', () => {});
+
+	describe('_computeBestPeer()', () => {
```

Comment:
current convetion is
`#_computeBestPeer` I think

Reply:
Perhaps we can add this to Notion, as I have seen other tests using this format.